docs/get-started/testnets-and-devnets.md

```diff
@@ -43,32 +33,43 @@ You can download the current Cardano blockchain network configuration files for
 
 #### Pre-Production Testnet
 
-- The pre-production testnet is a testing environment used to validate major upgrades and releases before deployment to the mainnet.
+- The Pre-Production Testnet is a testing environment used to validate major upgrades and releases before deployment to the mainnet.
 - It is a staging area where developers can simulate real-world scenarios and ensure that everything is working as expected before going live.
 - The preprod testnet helps to minimize the risk of bugs, security issues, and other problems that could negatively impact the mainnet.
 - Hard forks at approximately the same time as mainnet (within an epoch of each other)
 
-You can download the current Cardano blockchain network configuration files for Pre-production Testnet here: [The Cardano Operations Book > Pre-production Testnet.](https://book.world.dev.cardano.org/environments.html#pre-production-testnet)
+You can download the current Cardano blockchain network configuration files for Pre-Production Testnet here: [The Cardano Operations Book > Pre-Production Testnet.](https://book.world.dev.cardano.org/environments.html#pre-production-testnet)
 
 #### Summary
 
-The pre-production testnet is a more comprehensive testing environment used to validate major upgrades, while the preview testnet is a more targeted testing environment used to showcase new features and gather feedback from the community.
+The Pre-Production Testnet is a more comprehensive testing environment used to validate major upgrades, while the Preview Testnet is a more targeted testing environment used to showcase new features and gather feedback from the community.
 
 ### Where to get a testnet wallet?
-- [Daedalus Wallet for Cardano Testnet](https://testnets.cardano.org/en/testnets/cardano/get-started/wallet/) is the testnet version of Daedalus wallet.
-- [Nami Wallet](https://namiwallet.io/) is a lightweight wallet developed by Cryptonomic and supports both mainnet and testnet. 
-- [Ledger Nano S and Ledger Nano X](https://www.ledger.com/) is a hardware wallets that support both mainnet and testnet.
+- [Daedalus Wallet for Cardano Testnet](https://testnets.cardano.org/en/testnets/cardano/get-started/wallet/) is the Pre-Production and Preview versions of Daedalus wallet.
+- [Nami Wallet](https://namiwallet.io/) is a lightweight wallet developed by Cryptonomic and supports both Pre-Production and Preview testnets. 
```

Comment:
Can we add Eternl here, so devs on testnets have both a single-address and multi-address lightweight wallet option?  It's had Preprod / Preview support for about 6 months now (https://twitter.com/eternlwallet/status/1560671375987052545?lang=en)

Reply:
OK, have added it 🤓